src/provisioner/cluster.py

```diff
@@ -58,13 +60,18 @@ def _validate(self, node: dict):
         node_name = node.get('name')
         if node_name is None:
             raise CortxProvisionerError(errno.EINVAL, 'Missing name for the node entry')
+
+        PLog.debug("Validating node '%s' properties" % node_name)
```

Comment:
this should be info level, as its very useful while debugging

Reply:
W.r.t cortx-setup command action INFO and DEBUG level are defined. Validation happens internally(**_validate**) and so put DEBUG here.
If still required to be INFO, I will bring it. Also please review the console message added in the PR message box.